### ISSUE TITLE
Invalid file format error message on while uploading the data files for collection monitors

### ISSUE THREAD

**Issue by sseenivasan89** [a85/POSTMan-Chrome-Extension#11151]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

when i upload the data file (csv) to the collection monitors, it says as **_Invalid file format. Only JSON and CSV files are accepted._** 

there is no error when i used the same file to run collections from collection runner and the collection is running fine

**Note** : json data file format is working fine in collection monitor

### Steps To Reproduce

1. Open the monitor tab
2. Create monitor
3. Upload valid csv file in Upload a data file section
4. You could see the error message as **_Invalid file format. Only JSON and CSV files are accepted_**

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/44080743/182979906-ca93b573-9122-4156-b384-119760e71ada.png)

![image](https://user-images.githubusercontent.com/44080743/182979948-41396683-0769-49b9-9539-ee96cfd85699.png)

### Operating System

Windows

### Postman Version

9.27.1

### Postman Platform

Postman App

**Comment by DannyDainton** [comment]
Hey @sseenivasan89 

Can you supply a copy of the file that you're seeing the issue with, please? Also, what does the preview of the Runner file look like when opened?

**Comment by sseenivasan89** [comment]
Hey @DannyDainton 

when i tried with collection runner the preview was as expected but when i uploaded the same file in monitor it showed me the above error
attaching the file what i tried 

[DataMonitor.csv](https://github.com/postmanlabs/postman-app-support/files/9267682/DataMonitor.csv)

**Comment by DannyDainton** [comment]
I've been unable to reproduce this issue with that file - It will load fine in both the Runner and Monitor datafile sections. 

I did see the error when I renamed the file type when selecting it in the file explorer. 

![image](https://user-images.githubusercontent.com/17089546/183097717-b1d1f549-4710-460c-9477-234d44dd1112.png)

Was your file originally saved as something else, like an `.xls` before changing it to a `.csv` - Just trying to think of ways that this could occur. Other non valid file types _should_ be disabled in the explorer view when selecting them.
